config/500-webhook-deployment.yaml

```diff
@@ -82,6 +82,25 @@ spec:
           seccompProfile:
             type: RuntimeDefault
 
+        readinessProbe:
+          periodSeconds: 1
+          httpGet:
+            scheme: HTTPS
+            port: 8443
+            httpHeaders:
+              - name: k-kubelet-probe
+                value: "webhook"
+          failureThreshold: 3
+        livenessProbe:
+          periodSeconds: 1
+          httpGet:
+            scheme: HTTPS
+            port: 8443
+            httpHeaders:
+              - name: k-kubelet-probe
+                value: "webhook"
+          failureThreshold: 6
+
```

Comment:
One more thing, probably we should add `initialDelaySeconds: 20`? I think https://github.com/knative/serving/pull/9661 makes sense and other webhook have it.